example.yml

```
@@ -0,0 +1,230 @@
+            VERSION=${{ steps.params.outputs.VERSION }}
+            
+      - name: Extract host without port
+        id: parse_host
+        run: |
+          # Remove port from SERVER_HOST if present
+          HOST="${{ secrets.SERVER_HOST }}"
+          # Extract just the hostname/IP, removing any :port suffix
+          CLEAN_HOST=$(echo "$HOST" | cut -d':' -f1)
+          echo "CLEAN_HOST=$CLEAN_HOST" >> $GITHUB_OUTPUT
+          echo "📋 Using host: $CLEAN_HOST"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)